pinot-spi/src/main/java/org/apache/pinot/spi/filesystem/PinotFS.java

```diff
@@ -108,8 +123,10 @@ boolean doMove(URI srcUri, URI dstUri)
    * @return true if copy is successful
    * @throws IOException on IO failure
    */
-  boolean copy(URI srcUri, URI dstUri)
-      throws IOException;
+  default boolean copyDir(URI srcUri, URI dstUri)
```

Comment:
Let's not adding default for `copyDir()`. All the implementation should have it implemented